client/.eslintrc.js

```diff
@@ -24,17 +24,6 @@ module.exports = {
           ['^(~shared)(/.*|$)'],
           ['^(~)(/.*|$)'],
           ['^(~typings)(/.*|$)'],
```

Comment:
nit: i think line 25 and 26 can be removed too since they are unused for now

Reply:
OK removed!